packages/magento-customer/components/ForgotPasswordForm/ForgotPasswordForm.tsx

```diff
@@ -1,70 +1,43 @@
-import { Button, Form, FormActions, FormRow } from '@graphcommerce/next-ui'
-import { emailPattern, useFormGqlMutation } from '@graphcommerce/react-hook-form'
-import { i18n } from '@lingui/core'
+import { Button, Form, FormActions } from '@graphcommerce/next-ui'
+import { FormProvider, useFormGqlMutation } from '@graphcommerce/react-hook-form'
 import { Trans } from '@lingui/react'
-// eslint-disable-next-line @typescript-eslint/no-restricted-imports
-import { TextField, Alert, SxProps, Theme } from '@mui/material'
-import { ApolloCustomerErrorAlert } from '../ApolloCustomerError/ApolloCustomerErrorAlert'
+import { SxProps, Theme } from '@mui/material'
+import { PropsWithChildren } from 'react'
+import { EmailField } from '../CustomerFields/EmailField'
 import {
   ForgotPasswordDocument,
   ForgotPasswordMutation,
   ForgotPasswordMutationVariables,
 } from './ForgotPassword.gql'
 
-export function ForgotPasswordForm(props: { sx?: SxProps<Theme> }) {
-  const { sx = [] } = props
+export function ForgotPasswordForm(props: { sx?: SxProps<Theme> } & PropsWithChildren) {
+  const { sx = [], children } = props
   const form = useFormGqlMutation<ForgotPasswordMutation, ForgotPasswordMutationVariables>(
     ForgotPasswordDocument,
   )
-  const { muiRegister, handleSubmit, required, data, formState, error } = form
+  const { handleSubmit, formState } = form
   const submitHandler = handleSubmit(() => {})
 
-  if (formState.isSubmitSuccessful && data) {
-    return (
-      <Alert
-        severity='success'
-        variant='standard'
-        sx={(theme) => ({
-          marginTop: theme.spacings.md,
-          marginBottom: theme.spacings.sm,
-        })}
-      >
-        <Trans id='We’ve send a password reset link to your email address!' />
-      </Alert>
-    )
-  }
-
```

Comment:
Should this message not be added back somewhere?

Reply:
Added it back